cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,97 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	var billingAccounts []string
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+
+	// get a list of CloudAccountID
+	for _, account := range cloudAccounts {
+		if account.CloudProviderID() == "aws" && !helper.HasPrefix(billingAccounts, account.CloudAccountID()) {
+			contracts, hasContract := account.GetContracts()
+			var contractString string
+			if hasContract {
+				dimensions := contracts[0].Dimensions()
+				for _, dimension := range dimensions {
+					if ocm.GetDimensionValue(dimension, "four_vcpu_hour") > 0 ||
+						ocm.GetDimensionValue(dimension, "control_plane") > 0 {
+						contractString = " [Contract enabled]"
+					}
+				}
+			}
+			billingAccounts = append(billingAccounts, account.CloudAccountID()+contractString)
+		}
+	}
+
+	if billingAccount == "" && !interactive.Enabled() {
```

Comment:
I feel like this whole block should be moved above the data massaging of cloudAccounts above. 
This block doesn't require the above logic besides a tiny bit of optimization.
Even if we want the optimization I think we can just separate out the aws filter into another helper function or loop.

Reply:
@mikeshng I moved it around a bit, would you mind taking another look and see if it's where you expected?

Reply:
Looks good.